### DISCUSSION TITLE
Updates of downloads

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#86] [Q&A] [answered]
We can notify about updates of downloads of offline content: maps, poi, charts.
Currently users can check the updates tab on the download screen.

It could be via an Android notification or simpler like other apps:
show the number of updates (in brackets) in app menu screens.

**Comment by WolfgangZ42** [comment]
Not really necessary - but nice!

**Reply by airald** [reply]
same oppinion

**Comment by devemux86** [comment]
Cruiser will notify you when there are updates to your downloads (maps, poi, graphs):
- Menu show the number of updates: (4), (2), ...
- With a message

Info: BRouter is updated daily!

<img src="https://github.com/devemux86/cruiser/assets/3484020/de3e2080-39f2-46af-8afc-f10e8eb512ee" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/af6b59fb-0f8d-4976-9b30-a54ddf136d1a" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c16c14a2-7cad-4d5f-91cd-bf8913ef9373" width="33%"/>

**Comment by devemux86** [comment, marked as answer]
Notification of updates of your downloads is available in [Cruiser 3.0.21](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by devemux86** [reply]
Downloading Mapsforge maps also downloads the Mapsforge poi for the map,
so offline search is immediately available in the regions you have downloaded.

**Comment by devemux86** [comment]
An improvement for the downloads will be available in next Cruiser:

In local downloads (2nd) screen:
- **Tap**: switch on / off quickly the downloaded maps / poi
(similar to favorites)
- **Long press**: menu with extra actions like delete, etc.

**Reply by devemux86** [reply]
Available in [Cruiser 3.0.30](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
works
